src/main/frontend/db/query_react.cljs

```diff
@@ -126,5 +126,7 @@
         (pprint "query-opts:" query-opts)
         (apply react/q repo k query-opts query inputs))
       (catch js/Error e
+        (when throw-exception
+          (throw (ex-info (.-message e) {})))
```

Comment:
It could help if all callers default to this behavior to make failure more explicit to the user and to us. I didn't do that as I didn't want to change other features without understanding them